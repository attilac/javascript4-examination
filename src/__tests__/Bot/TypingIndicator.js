import React from 'react';
import { shallow } from 'enzyme';
import TypingIndicator from '../../components/Bot/TypingIndicator';

it('should return null by default', ()=>{
  const wrapper = shallow(<TypingIndicator />)
  expect(wrapper.html()).toBeFalsy();
})

it('should return null when called with false', ()=> {
  const wrapper = shallow(<TypingIndicator typing={false} />)
  expect(wrapper.html()).toBeFalsy();
})

it('should match snapshot when called with true', ()=> {
  const wrapper = shallow(<TypingIndicator typing />)
  expect(wrapper.html()).toMatchSnapshot();
})