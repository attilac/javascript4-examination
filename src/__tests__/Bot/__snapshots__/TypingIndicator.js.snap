// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot when called with true 1`] = `"<div class=\\"TypingIndicator bg-indigo-dark text-white font-bold px-4 h-8 rounded-full\\"><span></span><span></span><span></span></div>"`;
